polaris-react/src/components/EmptyState/EmptyState.stories.tsx

```diff
@@ -70,3 +70,19 @@ export function WithFullWidthLayout() {
     </Card>
   );
 }
+
+export function WithoutContentContainer() {
```

Comment:
added this story for any cases when an empty state component is not wrapped in a content container